static/src/stylesheets/module/content/_recipe-article.scss

```diff
@@ -0,0 +1,439 @@
+
+$grey:#dbdbdb;
+$grey-two:#f6f6f6;
+$grey-three:#767676;
+$purple:#B72166;
+
+$padding-wide: 100px;
```

Comment:
I know its weird, but could you use values from our grid system instead https://github.com/guardian/guss-grid-system?